Services/service_utilities.c

```diff
@@ -1,17 +1,41 @@
-#include "service_utilities.h"
+/*
+ * Copyright (C) 2015  University of Alberta
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+/**
+ * @file service_utilities.c
+ * @author upSat, Andrew Rooney
+ * @date 2020-06-06
+ */
 
-#include "time_management_service.h"
+#include "service_utilities.h"
 
-#undef __FILE_ID__
-#define __FILE_ID__ 5
+#include <stdarg.h>
+#include <stdio.h>
 
-struct _pkt_state {
-  uint8_t seq_cnt[LAST_APP_ID];
-};
+#include "time_management_service.h"
 
-static struct _pkt_state pkt_state;
+void ex2_log(const char *format, ...) {
+  va_list arg;
+  va_start(arg, format);
+  fprintf(stdout, format, arg, 0);
+  va_end(arg);
+  return;
+}
```

Comment:
What does this do?

Reply:
Currently it just wraps printf, but my thinking was that we should start using our own logging function sooner rather than later so when we want to replace prints with logs to a file, or to a serial port etc. then we only have to change this function.